### PR TITLE
Add using declaration terminology

### PR DESCRIPTION
**Pull request by gewarren**
cc @BillWagner

### CODE REVIEW

File: docs/fundamentals/code-analysis/style-rules/ide0063.md [hunk 1]
```diff
@@ -26,14 +26,15 @@ dev_langs:
 
 ## Overview
 
-This style rule concerns the use of `using` statements without curly braces. This alternative syntax was introduced in C# 8.0.
+This style rule concerns the use of `using` statements without curly braces, also known as `using` declarations. This [alternative syntax](~/_csharplang/proposals/csharp-8.0/using.md) was introduced in C# 8.0.
+
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/style-rules/ide0063.md:30]:
```suggestion
```

**Comment by gewarren** [comment]
Ping @BillWagner.

**Review by BillWagner — APPROVED** [review]
thanks for the ping @gewarren I'm not sure how I missed this.

This LGTM. You can :shipit: when ready.

**Comment by gewarren** [comment]
Thanks for the review @BillWagner!

**Pull request merged**